gandi/cli/commands/config.py

```diff
@@ -0,0 +1,67 @@
+""" Configuration namespace commands. """
+
+import sys
+import os
+import subprocess
+
+import click
+
+from gandi.cli.core.cli import cli
+from gandi.cli.core.params import pass_gandi
+
+
+@cli.command()
+@click.option('-g', help='Get from global configuration only'
+                         '(default=env -> local -> global).',
+              is_flag=True, default=False)
+@click.argument('key')
+@pass_gandi
+def get(gandi, g, key):
+    """Display value of a given config key."""
+    val = gandi.get(key=key, global_=g)
+    if not val:
+        gandi.echo("No value found.")
+        sys.exit(1)
+    gandi.echo(val)
+
+
+@cli.command()
+@click.option('-g', help='Edit global configuration (default=local).',
+              is_flag=True, default=False)
+@click.argument('key')
+@click.argument('value')
+@pass_gandi
+def set(gandi, g, key, value):
+    """Update or create config key/value."""
+    gandi.configure(global_=g, key=key, val=value)
+
+
+@cli.command()
+@click.option('-g', help='Edit global configuration (default=local).',
+              is_flag=True, default=False)
+@pass_gandi
+def edit(gandi, g):
+    """Edit config file with prefered text editor"""
+    config_file = gandi.home_config if g else gandi.local_config
+    path = os.path.expanduser(config_file)
+    editor = gandi.get('editor', 'vi')
```

Comment:
If the env var is checked, this is great.

Using vi to "simplify" sounds wrong to me.

If you never used vi you will have no idea of how to edit the file.
Some editors like nano are more friendly for newbies, but some people
do not like it too. So asking for the user to set the EDITOR looks great to me.